api/model.py

```diff
@@ -22,14 +22,9 @@ def get(self):
         return MODEL_META_DATA
 
 
-pred_txt = api.model('LabelPrediction', {
-    'pred_txt': fields.String(required=True, description='List containing the generated text')
-})
-
-
 predict_response = api.model('ModelPredictResponse', {
     'status': fields.String(required=True, description='Response status message'),
-    'pred_txt': fields.List(fields.Nested(pred_txt), description='Generated text based on input')
+    'pred_txt': fields.String(required=True, description='Generated text based on input')
```

Comment:
What was this a List before?

Reply:
Yes, it was a list of a list which didn't make any sense. Instead of calling `response['pred_txt'][0]['pred_txt']` to get the response you can just call `response['pred_txt']`

Reply:
See #16